templates/modular/lightslider.html.twig

```diff
@@ -11,73 +11,86 @@
-        {% for panel in page.content|split('<hr />') %}
-            {% set panel_media = page.media['image-'~loop.index~'.jpg'] %}
-            {% set panel_image = panel_media.brightness(settings.type_text_brightness|default('-100')).url %}
-            {% if panel_media and settings.gallery %}
-                {% set panel_data_thumb = 'data-thumb=\"'~panel_media.cropResize(settings.gallery_thumb_width|default(100),settings.gallery_thumb_height|default(100)).url~'\"' %}
-            {% endif %}
-            <li {{ panel_data_thumb }}>
-                <div class="panel-bg" style="
-                    {% if panel_image %}background-image: url({{ panel_image }});{% endif %}
-                    min-height: {{ settings.type_text_min_height|default('auto') }};
-                    height: {{ settings.type_text_height|default('auto') }};
+
+      {% for panel in page.content|split('<hr />') %}
+        {% set panel_media = page.media['image-'~loop.index~'.jpg'] %}
+        {% set panel_image = panel_media.brightness(settings.type_text_brightness|default('-100')).url %}
+        {% if panel_media and settings.gallery %}
+            {% set panel_data_thumb = 'data-thumb=\"'~panel_media.cropResize(settings.gallery_thumb_width|default(100),settings.gallery_thumb_height|default(100)).url~'\"' %}
+        {% endif %}
+
+        <li {{ panel_data_thumb }}>
+            <div class="panel-bg" style="
+                {% if panel_image %}background-image: url({{ panel_image }});{% endif %}
+                min-height: {{ settings.type_text_min_height|default('auto') }};
+                height: {{ settings.type_text_height|default('auto') }};
+                ">
+                <div class="panel-padding {{ settings.type_text_css_class }}" style="
+                    padding: {{ settings.type_text_vertical_padding|default(0) }} {{ settings.type_text_horizontal_padding|default(0) }}
                     ">
-                    <div class="panel-padding {{ settings.type_text_css_class }}" style="
-                        padding: {{ settings.type_text_vertical_padding|default(0) }} {{ settings.type_text_horizontal_padding|default(0) }}
-                        ">
-                        {{ panel }}
-                    </div>
+                    {{ panel }}
                 </div>
-            </li>
-        {% endfor %}
+            </div>
+        </li>
+
+      {% endfor %}
+
     </ul>
-    {% else %}
+
+  {% else %}
+
     <ul id="{{ unique_id }}">
-        {% for image in page.media.images %}
-            {% if settings.gallery %}
-                {% set image_data_thumb = 'data-thumb=\"'~image.cropResize(settings.gallery_thumb_width|default(100),settings.gallery_thumb_height|default(100)).url~'\"' %}
-            {% endif %}
-            <li {{ image_data_thumb }}>
-                <img src="{{ image.url }}" />
-            </li>
-        {% endfor %}
+
+      {% for slide in page.header.slides %}
```

Comment:
The problem is this introduces a breaking change, and we don't want breaking changes as people updating the plugin without knowing the change will have their site broken. It must continue to work as before _and_ provide additional functionality. It can check if page.header.slides is defined, otherwise use page.media.images as before.